bin/faucet/src/faucet/mod.rs

```diff
@@ -231,31 +231,28 @@ impl Faucet {
             genesis_chain_mmr,
         ));
 
-        let public_key = match &account_file.auth_secret_key {
-            AuthSecretKey::RpoFalcon512(secret) => secret.public_key(),
-        };
-
-        let authenticator = Arc::new(BasicAuthenticator::<StdRng>::new(&[(
-            public_key.into(),
-            account_file.auth_secret_key,
-        )]));
+        let keypairs = account_file
+            .auth_secret_keys
+            .into_iter()
+            .map(|key| match key {
+                AuthSecretKey::RpoFalcon512(ref falcon) => (falcon.public_key().into(), key),
+            })
+            .collect::<Vec<_>>();
+        let authenticator = BasicAuthenticator::<StdRng>::new(&keypairs);
```

Comment:
Should we be enforcing that there is one or at least one authentication key? I'm unsure what this implies if its empty, or multiple.

Reply:
For the current usage, I think we should enforce that there is exactly one (if there is none or more, probably implies a bug somewhere).